tests/bootstrap.php

```diff
@@ -23,4 +30,40 @@
 $root = $findRoot(__FILE__);
 unset($findRoot);
 chdir($root);
-require $root . '/vendor/cakephp/cakephp/tests/bootstrap.php';
+
+require_once 'vendor/autoload.php';
+
+define('ROOT', $root . DS . 'tests' . DS . 'test_app' . DS);
```

Comment:
There's no need to complicate the bootstrapping when just `require $root . '/vendor/cakephp/cakephp/tests/bootstrap.php';` works.

Reply:
I was just used to this kind of bootstrapping mechanism to get it working but I have now reverted it.

Reply:
For most plugins this simple loading is enough, only few need additional/custom config.